.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+name: Test
+
+on:
+  - push
+  - pull_request
+  - workflow_call
+
+jobs:
```

Comment:
I don't think this is the same as the existing tests in `.circleci/config.yml`. The tests will install the package in 3 different ways (source, source with `--pre`, and sdist), and in each of them run the unittest to make sure things can import and run. 
However I think the test here will just install the package and run the tests once.

Reply:
I left out calling `--pre` on source because I felt it didn't make sense, but if you look at the if statement, it tests properly for source and sdist.